hetmech/degree_weight.py

```diff
@@ -137,6 +137,7 @@ def dwwc(graph, metapath, damping=0.5, dense_threshold=0, dtype=numpy.float64):
         converted to a dense automatically.
     dtype : dtype object
     """
+    metapath = graph.metagraph.get_metapath(metapath)
```

Comment:
We shouldn't need this because the decorator should already be doing this conversion?

Reply:
Good call. Do you think we will want to put the decorator on the chain function as well?